README.md

```diff
@@ -40,6 +40,7 @@ make dev.checkout
 make dev.clone
 make dev.provision
 echo export OPENEDX_RELEASE=hawthorn.master > .envrc
+direnv allow
```

Comment:
This migh work for you personally, but not everyone uses direnv, so I think it would create more clutter. Ironically